### PR TITLE
Removing content api

### PR DESCRIPTION
**Pull request by carvil**
We used to get licences from the Content API by their GDS IDs. This is
basically a search call that the Content Store does not support.

Since we are retiring the Content API and replace it with modern
services and APIs, we decided that using Rummager to fetch licence
metadata (licence identifiers and short descriptions) is an acceptable
replacement.

In this PR we:
- Remove unused call to the Content API to fetch the `/licence-finder` artefact;
- Upgrade `gds-api-adapters` so that we can use the `search` endpoint in Rummager;
- Replace calls to `Services.content_api.licences_for_ids` with a call to search in Rummager.

Licences now have both `licence_identifier` and `licence_short_description` fields in search. here is an example: https://www.gov.uk/api/search.json?filter_link=/food-business-registration&fields=title,link,licence_identifier,licence_short_description

Trello: https://trello.com/c/vlqSHmTZ/76-make-licence-finder-use-something-else-to-fetch-licenses

### CODE REVIEW

File: app/models/licence_facade.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class LicenceFacade
   def self.create_for_licences(licences)
     api_data = get_licence_artefacts(licences)
     licences.map do |l|
-      matching_api_data = api_data['results'].find { |d| l.gds_id.to_s == d['details']['licence_identifier'] }
+      matching_api_data = api_data['results'].find { |d| l.gds_id.to_s == d['licence_identifier'] }
```
Comment (alecgibson) [inline app/models/licence_facade.rb]: (Eww... single-letter variable names)

File: app/models/licence_facade.rb [hunk 2]
```diff
@@ -38,10 +41,16 @@ def title
   end
 
   def url
-    published? ? @artefact['web_url'] : nil
+    published? ? web_url : nil
   end
 
   def short_description
-    published? ? @artefact['details']['licence_short_description'] : nil
+    published? ? @artefact['licence_short_description'] : nil
```
Comment (alecgibson) [inline app/models/licence_facade.rb]: If we're retiring use of Content API, should we also remove all references to `@artefact`?

File: app/models/licence_facade.rb (hunk 1 above)
Comment (carvil) [inline app/models/licence_facade.rb]: Yeah, I didn't want to change things too much so it was clearer from the diff what it takes to replace the Content API calls, but I will fix this one as it's probably ok.

File: app/models/licence_facade.rb (hunk 2 above)
Comment (carvil) [inline app/models/licence_facade.rb]: I'll rename this to be more appropriate.

**Pull request merged**